language/move-lang/tests/move_check/liveness/loop_weirdness.exp

```diff
@@ -0,0 +1,8 @@
+error: 
+
+   ┌── tests/move_check/liveness/loop_weirdness.move:9:36 ───
+   │
+ 9 │                 if (my_local >= 0) { break; };
+   │                                    ^^^^^^^^^^ Unreachable code. This statement (and any following statements) will not be executed. In some cases, this will result in unused resource values.
```

Comment:
Only question I have is with the span of this error reporting; shouldn't it be only on the part including/after the `;` and not on the break itself? But I'm not even fully convinced that's a good idea, more just something to think about.

Reply:
It should point to just the `;`. 

The issue is we don't track the location of the last semicolon currently, which makes the location of the `()` inserted inaccurate. 

@bob-wilson, we should fix this

Reply:
Is there an AST to attach the span for that? It's a Sequence that ends with a boxed optional expression, and in this case the expression is missing and the optional is None. Do you want to change the AST to something different?

Reply:
I discussed this with Todd. I'll add a new field to the AST to record the semicolon location.